prac_08/squaring.py

```diff
@@ -0,0 +1,34 @@
+"""
+CP1404/CP5632 Practical
+Kivy GUI program to square a number
+Lindsay Ward, IT@JCU
+Started 13/10/2015
+"""
+
+from kivy.app import App
+from kivy.lang import Builder
+from kivy.core.window import Window
+
+__author__ = 'Lindsay Ward'
+
+
+class SquareNumberApp(App):
+    """ SquareNumberApp is a Kivy App for squaring a number """
+    def build(self):
+        """ build the Kivy app from the kv file """
+        Window.size = (200, 100)
+        self.title = "Square Number"
+        self.root = Builder.load_file('squaring.kv')
+        return self.root
+
+    def handle_calculate(self):
+        """ handle calculation (could be button press or other call), output result to label widget """
+        input_text = self.root.ids.input_number.text
```

Comment:
Should we have passed the input value in as a parameter to the function rather than getting it within the function?